statements/downloaded_file.md

```diff
@@ -0,0 +1,35 @@
+# Downloaded a file
+
+## Description
+
+This event is emitted when a user has downloaded a file such as a pdf, doc, txt, ppt, xls, csv, etc.
+
+## Example
+
+```json
+{
```

Comment:
I would recommend to add the timestamp (cf. [this discussion](https://github.com/gaia-x-dases/xapi-virtual-classroom/discussions/8) we had on the virtual classroom profile)